### DISCUSSION TITLE
Volume name display "/" on macOS 14.4

### DISCUSSION THREAD

**Discussion by Stingraphic** [iterate-ch/cyberduck#15698] [Q&A] [answered]
Volume name display "/" in macOS 14.4, but the folder path is still reachable, Is it a App bug or macOS bug?
![WX20240308-130706@2x](https://github.com/iterate-ch/cyberduck/assets/9998319/b36b7f62-0036-4641-ab41-21efd7030d11)

**Comment by dkocher** [comment, marked as answer]
We will release a bugfix release later today with a workaround for this problem.

**Reply by Stingraphic** [reply]
Thank you!

**Comment by dkocher** [comment]
We have reported this regression in macOS 14.4 beta builds to Apple (FB13592916) a month ago but unfortunately (as usual with this black hole) received no acknowledgment of the bug. The current snapshot build includes a workaround we have implemented to fix the display in Finder.app.

**Comment by Stingraphic** [comment]
> We have reported this regression in macOS 14.4 beta builds to Apple (FB13592916) a month ago but unfortunately (as usual with this black hole) received no acknowledgment of the bug. The current snapshot build includes a workaround we have implemented to fix the display in Finder.app.

Hi, i just update to the latest version 4.15.4 (21882), the volume path ended with ".localized", is it normal? Is there any way to remove the ".localized". Thanks.

**Reply by dkocher** [reply]
We have changed the mountpoint to a directory with the `.localized` suffix to workaround the display issue in Finder.app.

**Reply by Stingraphic** [reply]
I think there is a more correct way to fix this problem, now it even not showing on Locations. Hope can fix it soon.

**Reply by dkocher** [reply]
See #15703.

**Reply by sir-oliver** [reply]
This newly added ending ".localized" broke all of our VSCode projects files and automated build tools. 
Will this stay or is it just a temporary workaround? I don't want to manually edit  100+ VSCode projects and build tools just to find out, that with the next release it is all back to how it was.
Thanks!

**Reply by dkocher** [reply]
We will see if it is feasible to create a symbolic link that points from the previous location to the new mount point with the `.localized` suffix.

**Reply by cedrickp32** [reply]
The last update fix the display in Finder. I use applescript to automate some tasks, and when i get the path it still show as /: so i get errors can't make into type alias. Any fix for something like that ?

**Reply by cedrickp32** [reply]
@dkocher Is there anything to do ?

**Reply by dkocher** [reply]
Unfortunately this is caused by a bug in macOS 14.4.

**Comment by apastuszak** [comment]
I installed the update and now my mounted providers are not showing up on my desktop or in Finder under locations. Is there a way to fix this?

**Reply by dkocher** [reply]
See #15703 to revert the display of mounted volumes on the Desktop and in Finder → Network.

**Comment by apastuszak** [comment]
When Apple fixes this will Mountain Duck "just work," or will you need to push another update?

**Reply by dkocher** [reply]
If Apple provides a fix we can release an update that will revert the default to show mounted volumes on Desktop.

**Comment by apastuszak** [comment]
Does 14.4.1 that just got released fix this?

**Reply by Stingraphic** [reply]
Same question.

**Reply by dkocher** [reply]
macOS 14.4.1 does not include any fix.

**Reply by cedrickp32** [reply]
Indeed, i tested with my problem and i still get the / at the beginning
